apps/full-site-editing/full-site-editing-plugin/dotcom-fse/blocks/navigation-menu/index.js

```diff
@@ -12,10 +12,12 @@ import edit from './edit';
 import './style.scss';
 
 const icon = (
-	<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
-		<path fill="none" d="M0 0h24v24H0V0z" />
-		<path d="M12 7.27l4.28 10.43-3.47-1.53-.81-.36-.81.36-3.47 1.53L12 7.27M12 2L4.5 20.29l.71.71L12 18l6.79 3 .71-.71L12 2z" />
-	</svg>
+	<img>
```

Comment:
Any particular reason this was updated? :) Shouldn't be an issue, but the code path isn't really used any more

Reply:
Woopsie, nop! I added this to test something and forgot to remove. Thanks for catching!